docs/build/quick-start/add-zksync-to-metamask.md

```diff
@@ -36,4 +36,14 @@ Fill in the following details for the zkSync network:
 
 Click on "Save" to add the zkSync network to MetaMask.
 
+## Import from Chainlist
+
+To add zkSync as a custom network in Metamask from Chainlist.org, follow these steps:
+
+- Visit https://chainlist.org/chain/324 (Mainnet) or https://chainlist.org/chain/300 (Testnet) on your browser
+- Click Connect
+- Unlock your metamask
+- Show metamask popup `Allow this site to add a network?` then click Approve.
```

Comment:
For instructions, we should write them from the perspective of the user. With that in mind, thoughts about the following edit?

Note: Also remove the ending period, `.`, for consistency
```suggestion
- When prompted with `Allow this site to add a network?`, click `Approve`
```